frontend/packages/console-dynamic-plugin-sdk/src/extensions/add-actions.ts

```diff
@@ -2,6 +2,9 @@ import * as React from 'react';
 import { Extension, ExtensionDeclaration, CodeRef } from '../types';
 import { AccessReviewResourceAttributes } from './console-types';
 
+/** Allows plugins to contribute an add action item to the add page of developer perspective.
```

Comment:
Are these the same as above?

Reply:
Updated the `dev-console.add/action-group` so it does make more sense.